docs/3.x upgrade guide.md

```diff
@@ -0,0 +1,15 @@
+# 3.x Upgrade Guide
+
+## BREAKING
+
+### Bring your own polyfills
+
+core-js@2 has been removed from @rjsf/core. See more about [the rationale here](https://github.com/rjsf-team/react-jsonschema-form/pull/2211#issue-563700810).
+
+If you're using a framework like [Create React App](https://create-react-app.dev/docs/supported-browsers-features#supported-browsers), [Gatsby](https://www.gatsbyjs.com/docs/how-to/custom-configuration/browser-support/), or [Next.js](https://nextjs.org/docs/basic-features/supported-browsers-features), polyfills are already included there.
+
+If you were directly depending on @rjsf/core's @babel/runtime pulling in core-js@2, [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env#how-does-it-work) is probably a good first choice.
```

Comment:
Just want to clarify here -- is the only solution here that we have to begin transpiling the code with `@babel/preset-env`? What if we just ran `npm install core-js` and added `import 'core-js';` -- would that also be a solution?

Reply:
I added a note to that effect.